### PR TITLE
[WIP]: US276431 User services build and deletion in gitlab

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

Build deploy user services of 3 types into prod and dev environments
Delete user services

### Description of the Change
Made  changes in scm module and workflows

### Benefits

users can build deploy their services
users can delete their services

### CODE REVIEW

File: jazz-build-module/scm-module.groovy [hunk 1]
```diff
@@ -28,13 +27,15 @@ def setRepoEndpoints(){
         scm_user_services_clone_url = "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/"
         scm_branch_permission_api_endpoint = "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/rest/branch-permissions/2.0/projects/${scm_config.REPOSITORY.REPO_BASE_SERVICES}/repos/"
         serviceonboarding_repo	= "${scm_protocol}${scm_config.REPOSITORY.BASE_URL}/${scm_config.REPOSITORY.REPO_BASE_PLATFORM}/service-onboarding-build-pack.git"
-    }else if(scm_config.SCM.TYPE == "bitbucket"){
+		scm_webhook_target_url 	= "${scm_protocol}${scm_config.JENKINS.USERNAME}:${scm_config.JENKINS.JENKINS_PASSWORD}@${JenkinsLocationConfiguration.get().getUrl().split('/')[2]}/project/Gitlab-Trigger-Job"
```
Comment (suryajak) [inline jazz-build-module/scm-module.groovy]: We cannot have JenkinsLocationConfiguration come into SCM Module as we are no longer separating concerns.  The url value can be passed into the method "addWebhook" as a variable

File: jazz-build-module/scm-module.groovy [hunk 2]
```diff
@@ -166,22 +166,22 @@ def setRepoPermissions(repo_owner, repo_name, admin_group) {
     }
 }
 
-def addWebhook(repo_name, webhookName, targetUrl) {
+def addWebhook(repo_name, webhookName) {
```
Comment (suryajak) [inline jazz-build-module/scm-module.groovy]: targetUrl has to be a parameter, no passing Jenkins info in here.

File: jazz-build-module/scm-module.groovy (hunk 1 above)
Comment (SiniWilson) [reply jazz-build-module/scm-module.groovy]: In cooperated the review comment

File: jazz-build-module/scm-module.groovy (hunk 2 above)
Comment (SiniWilson) [reply jazz-build-module/scm-module.groovy]: In cooperated the review comment

**Pull request merged**